scripts/missions/windurst/2_3_0_The_Three_Kingdoms.lua

```diff
@@ -37,6 +37,7 @@ mission.reward =
 local handleAcceptMission = function(player, csid, option, npc)
     if option == 5 then
         mission:begin(player)
+        player:setMissionStatus(mission.areaId, 1)
```

Comment:
Why are you doing this here?  Nation missions require that no mission be active before a new one is began, and in this case missionStatus should be 0.  The only reason this would ever fail is if you set it as a GM without following the warning in the GM script.